src/clients/nfs-client4part.rb

```diff
@@ -54,6 +55,33 @@ def main
 
       nil
     end
+
+  private
+
+    # Generates the UI that allows to manage the NFS shares entries
+    #
+    # @return [Yast::Term] a term defining the UI
+    def create_ui
+      Wizard.SetHelpText(@help_text1)
+
+      ReplacePoint(ui_id, FstabTab())
+    end
+
+    # Updates the UI that allows to manage the NFS shares entries
+    #
+    # @return [Boolean] true when entries are successfully replaced; false otherwise.
+    def refresh_ui
+      # The UI could be not available yet as FromStorage action can be called just to sync
+      # the NFS entries before create the interface.
+      return unless UI.WidgetExists(ui_id)
+
+      UI.ReplaceWidget(ui_id, FstabTab())
+    end
+
+    # Returns the id for the NFS shares UI replace point
+    def ui_id
+      @id ||= Id(:fstab_rp)
```

Comment:
I would also avoid to use the attribute `@id`.

Reply:
Good catch, thanks